### PR TITLE
DX: Little cleanup

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -54,7 +58,7 @@ def up():
     """
     Ensure infrastructure is sync and running
     """
-    build()
+    # build()
```
Comment (pyrech) [inline fabfile.py]: Is that intended?

File: README.md [hunk 2]
```diff
@@ -127,18 +133,105 @@ Checkout `fab -l` to have the list of available fabric tasks.
...
 
-> ERROR: Service 'frontend' failed to build: pull access denied for app_basephp, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
+<summary>Read the cookbook</summary>
 
-In order to fix this issue, you can update the `services_to_build_first` variable
-in the `fabfile.py` file. This will force docker-compose to build theses
-services first.
+If you want to create a new Symfony project, you need to:
+
+1. Remove the `app` folder:
```
Comment (pyrech) [inline README.md]: Rename `app` into `application`

File: fabfile.py (hunk 1 above)
Comment (lyrixx) [reply fabfile.py]: nope, thanks

File: README.md [hunk 3]
```diff
@@ -13,11 +13,17 @@ why we created and open-sourced this starter-kit.
...
-  application
-* `env.project_hostnames`: This will be all your domain names, separated with comma
+* `env.project_name` (**required**): This will be used to prefix all docker
+objects (network, images, containers);
+
+* `env.root_domain` (optional, default: `project_name + '.test'`): This is the
+root domain where the application will be available;
+
+* `env.extra_domains` (optional): This contains extra domains where the
+application will be available
```
Comment (ternel) [inline README.md]:
```suggestion
application will be available;
```

File: README.md [hunk 4]
```diff
@@ -13,11 +13,17 @@ why we created and open-sourced this starter-kit.
...
+
+* `env.project_directory` (optional, default: `application`): This is the host
+directory containing your PHP application
```
Comment (ternel) [inline README.md]:
```suggestion
directory containing your PHP application.
```

**Pull request merged**